src/routes/docs/tutorials/+page.svelte

```diff
@@ -50,6 +50,17 @@
 							</p>
 						</a>
 					</li>
+					<li class="is-mobile-col-span-2">
```

Comment:
Let's leave it out, we'lll update this page later with all of the new tutorials

Reply:
Ok, I removed it. So now you travel to it by the URL?

Reply:
@timDeHof We will add it somewhere else, but we need to figure out how we'll do it. Dw about it, we'll handle it on our side :D